2-mandatory/2-water-bottle.js

```diff
@@ -19,20 +19,37 @@ let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    if(this.volume === 0){
+      return this.volume += 100
+    } else { return this.volume}
```

Comment:
Nice idea but what about if I want to fill up the bottle when it is half full (e.g. `volume` is `50`)?